e2e/vfolder.test.ts

```diff
@@ -1,11 +1,94 @@
-import { createVFolderAndVerify, deleteVFolderAndVerify, loginAsUser } from './test-util';
-
+import { expect, test } from '@playwright/test';
+import { createVFolderAndVerify, deleteVFolderAndVerify, loginAsAdmin, loginAsUser, loginAsUser2, logout } from './test-util';
 
 test.describe('VFolder ', () => {
   test('User can create and delete vFolder', async ({ page }) => {
     await loginAsUser(page);
     await createVFolderAndVerify(page, 'e2e-test-folder');
     await deleteVFolderAndVerify(page, 'e2e-test-folder');
   });
+
+  test('User can rename vfolder', async ({page}) => {
+    await loginAsUser(page);
+    await createVFolderAndVerify(page, 'e2e-rename-test-folder');
+    await page.waitForTimeout(1000);
+    await page.getByRole('button', { name: 'create' }).click();
+    await page.getByLabel('Type new folder name').click();
+    await page.getByLabel('Type new folder name').fill('renamed-e2e-test-folder');
+    await page.getByRole('button', { name: 'edit' }).click();
+    await page.locator('#input-vaadin-text-field-18').click();
+    await page.locator('#input-vaadin-text-field-18').press('ControlOrMeta+a');
+    await deleteVFolderAndVerify(page, 'renamed-e2e-test-folder');
+  })
+
+  test('User can share vFolder', async ({ browser }) => {
+    const page1 = await browser.newPage();
+    const page2 = await browser.newPage();
+
+    await loginAsUser(page1);
+    await createVFolderAndVerify(page1, 'e2e-share-test-folder');
+    await page1.waitForTimeout(1000);
+    //share vfolder with user
+    await page1.getByRole('button', { name: 'share' }).click();
+    await page1.getByLabel('Enter E-Mail address').click();
+    await page1.getByLabel('Enter E-Mail address').fill("user2@lablup.com");
+    await page1.getByLabel('Edit+Delete').check();
+    await page1.locator('#share-button').getByLabel('share').click();
+    //accept shared vfodler
+    await loginAsUser2(page2);
+    await page2.getByLabel('Accept').nth(0).click();
+    expect(page2.locator('.ant-notification-notice')).toBeTruthy();
+
+    await deleteVFolderAndVerify(page1, 'e2e-share-test-folder');
+  });
+
+  test('If a shared Vfolder is moved to the trash before accepting', async ({ browser }) => {
+    const page1 = await browser.newPage();
+    const page2 = await browser.newPage();
+
+    await loginAsUser(page1);
+    await createVFolderAndVerify(page1, 'e2e-share-test-folder');
+    await page1.waitForTimeout(1000);
+    //share vfolder with user
+    await page1.getByRole('button', { name: 'share' }).click();
+    await page1.getByLabel('Enter E-Mail address').click();
+    await page1.getByLabel('Enter E-Mail address').fill("user2@lablup.com");
+    await page1.getByLabel('Edit+Delete').check();
+    await page1.locator('#share-button').getByLabel('share').click();
+    //check vfolder invitation was sent
+    await loginAsUser2(page2);
+    await page2.waitForTimeout(1000);
+    expect(page1.getByTitle('Invitation').locator('#activity').nth(0));
```

Comment:
That code seems finds the element but doesn't validate any conditions like `toBeVisible`.
Also when there are multiple invitations, `nth(0)` seems to be a different folder invitation than e2e-share-test-folder.